### DISCUSSION TITLE
Integration with Shortcut

### DISCUSSION THREAD

**Discussion by SashaShikhmareva** [allure-framework/allure2#2768] [TestOps]
Hello, team!

Could you please let me know if you have considered integrating with the Shortcut bug tracker? I would love to see this integration for AllureTestOps.

Thanks in advance )

**Comment by cheshi-mantu** [comment]
hi there!
we've passed this request to the product team for the decision making.
Please consider filing ideas, feedback and bugfix requests to the dedicated support portal, where we could react faster  

cheers

**Reply by cheshi-mantu** [reply]
TODI-151
